.gitignore

```diff
@@ -0,0 +1,3 @@
+.venv
+pycache
+.idea
```

Comment:
Anyway .idea in the PR (
<img width="307" alt="image" src="https://github.com/mate-academy/py-taxi-service-startproject/assets/102595649/092cd2a0-8ce5-4512-a116-3427c10f633a">
<img width="307" alt="image" src="https://github.com/mate-academy/py-taxi-service-startproject/assets/102595649/092cd2a0-8ce5-4512-a116-3427c10f633a">